mdsuite/calculators/spatial_distribution_function.py

```diff
@@ -249,10 +249,9 @@ def run_calculator(self):
             subjects=["System"],
         )
 
-        if self.plot:
-            coordinates = tf.reshape(self._get_unit_sphere(), [self.args.n_bins ** 2, 3])
-            colour_map = tf.reshape(sdf_values, [-1])
-            self._run_visualization(coordinates, colour_map)
+        coordinates = tf.reshape(self._get_unit_sphere(), [self.args.n_bins ** 2, 3])
+        colour_map = tf.reshape(sdf_values, [-1])
+        self._run_visualization(coordinates, colour_map)
```

Comment:
Why remove the `plot=True` argument? The SDF does store data in the DB https://github.com/zincware/MDSuite/blob/244a4ca81d27599e81487ea51a2d81473683e8f0/mdsuite/calculators/spatial_distribution_function.py#L246-L247 so it should have a `plot=False` option?

Reply:
When plot is run it calls a method that the SDF does not have or want to have. We need to remove that or we can leave it at plot=False. As the SDF is less of a calculator and more visualisation tool I don't think it needs a plot method.